lib/screens/login.dart

```diff
@@ -71,25 +73,32 @@ class _LoginState extends State<Login> {
             ]),
             const SizedBox(height: 20),
             PlainButton(
-                text: "Connexion",
-                onPressed: () async {
-                  const url =
-                      'https://dvpm9zw6vc.execute-api.eu-west-3.amazonaws.com/auth/p/login';
-                  final response = await http.post(
-                    Uri.parse(url),
-                    headers: {'Content-Type': 'application/json'},
-                    body: jsonEncode({'email': email, 'password': password}),
+              text: "Connexion",
+              onPressed: () async {
+                SharedPreferences prefs = await SharedPreferences.getInstance();
+                const url =
+                    'https://dvpm9zw6vc.execute-api.eu-west-3.amazonaws.com/auth/p/login';
```

Comment:
It'll be better if the backend URL is specified inside a configuration file like for example the `.env` file